notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb

```diff
@@ -12,15 +12,7 @@
    "cell_type": "code",
```

Comment:
<p>This is my error.</p><p>"Monte Carlo Markov Chain are" should be is</p><p>Let's should be lets</p><p><br></p><p>Commenting so I remember to fix</p>

 _Reply via <a href='https://app.reviewnb.com/canyon289/bayesian-model-evaluation/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>